web/src/engine/websites/templates/Iken.ts

```diff
@@ -0,0 +1,71 @@
+import { FetchJSON } from '../../platform/FetchProvider';
+import { Chapter, DecoratableMangaScraper, type MangaPlugin, Manga } from '../../providers/MangaPlugin';
+import * as Common from '../decorators/Common';
+
+type APIMangas = {
+    posts: APIManga[]
+}
+
+type APISingleManga = {
+    post: APIManga
+}
+
+type APIManga = {
+    id: number,
+    slug: string,
+    postTitle: string,
+    chapters?: APIChapter[]
+}
+
+type APIChapter = {
+    id: number,
+    slug: string,
+    number: number,
+    title: string
+}
+
+type MangaID = {
+    id: string,
+    slug: string,
+}
+
+const pageScript = `[...document.querySelectorAll('section img[loading]')].map(image => new URL(image.getAttribute('src'), window.location.origin).href);`;
```

Comment:
Can be simplified by using property instead of attribute `[...].map(img => img.src);`